target_chains/ethereum/entropy_sdk/solidity/IEntropyConsumer.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.0;
+
+abstract contract IEntropyConsumer {
+    // This address should be set to point at the Entropy contract. If not set, the
+    // contract will revert when it receives a callback from Entropy.
+    // Entropy address can be found here - https://docs.pyth.network/entropy/contract-addresses
+    address public entropy;
```

Comment:
instead of adding to the contract state, we can create a virtual `getEntropy` function and use that in `_entropyCallback`. I think this can make it easier to integrate if all the contract state is stored somewhere else.